### DISCUSSION TITLE
`wire:dirty` not working with `wire:model.live`

### DISCUSSION THREAD

**Discussion by mtt786** [livewire/livewire#8227] [1. Help]
```
<form wire:submit="save">
   <input type="text" wire:model.live="name" />
   <p wire:dirty>Unsaved</p>
</form>
```
"Unsaved" appears and then disappears immediately.

**Comment by joshhanley** [comment]
@mtt786 dirty is only for properties that have changed on the front end that haven't yet been synced to the backend. As you are suing `.live`, changes are synced straight away, hence why it disappears immediately.

**Reply by mtt786** [reply]
Any solution to make it work with `.live` or alternate?

**Reply by joshhanley** [reply]
@mtt786 what do you mean? It's "working" as expected. What are you trying to achieve?

**Reply by andreas-freitag** [reply]
@joshhanley I think what he means is that there is no simple way to show to the user that a specific changed property was persisted in the backend. 

The wire:dirty directive is only shown (with .blur/.change longer than with .live) until the persistence happened, which might be quite short. A simple user of the app then has no indication if the persistence happened, except that there is no dirty indication.

We could show a general success message (as shown by Caleb in the screencasts) but that message is NOT property specific and does not allow for e.g. little green checkmarks inside a green-bordered input field which turns normal after a configurable duration.

Do you understand what I mean? And yes, my point might be a completely new and different discussion, I don't know :)

**Comment by nfauchelle** [comment]
I have ran into this just now, trying to work out why wire:dirty wasn't working with live.debouce.

I have a form which is auto saving, however I have set the debounce to 1 second for a big textarea / notes field.
Ideally I want to indicate to the user the save hasn't happended yet, and then flash when the save does fire.

Any reason not to support it?
